Marlin/language_en.h

```diff
@@ -299,6 +323,15 @@
   #ifndef MSG_UBL_SAVE_MESH
     #define MSG_UBL_SAVE_MESH                 _UxGT("Save Bed Mesh")
   #endif
+  #ifndef MSG_UBL_SAVE_ERROR
+    #define MSG_UBL_SAVE_ERROR                _UxGT("save_UBL_active() error")
+  #endif
+  #ifndef MSG_UBL_RESTORE_ERROR
+    #define MSG_UBL_RESTORE_ERROR             _UxGT("restore_UBL_active() error")
+  #endif
```

Comment:
Question: Shouldn't we stay below 20 characters? Only asking, because translating to german would be much easier having more space.